src/index.ts

```diff
@@ -321,9 +331,7 @@ export default function solidPlugin(options: Partial<Options> = {}): Plugin {
         inputSourceMap: false as any,
       };
 
-      if (id.includes('tsx')) {
-        opts.presets.push([ts, options.typescript || {}]);
-      }
```

Comment:
@high1 Can we keep this format? I prefer things to be explicit rather than one liners. It eases the readability in my opinion.

Reply:
I've did this because
if (!/\.[jt]sx/.test(id)) return null;
line 296